bugsnag-android-core/src/main/java/com/bugsnag/android/Configuration.kt

```diff
@@ -0,0 +1,330 @@
+     * bugsnag-android-ndk this flag is true by default.
+     */
+    var autoDetectNdkCrashes: Boolean = false
+
+    /**
+     * Sets whether Bugsnag should automatically capture and report unhandled errors.
+     * By default, this value is true.
+     */
+    var autoDetectErrors = true
+
+    /**
+     * Intended for internal use only - sets the code bundle id for React Native
+     */
+    var codeBundleId: String? = null
+
+    /**
+     * Intended for internal use only - sets the type of the notifier (e.g. Android, React Native)
+     */
+    var appType = "android"
+
+    /**
+     * Sets the delivery used to make HTTP requests to Bugsnag. A default implementation is
+     * provided, but you may wish to use your own implementation if you have requirements such
+     * as pinning SSL certificates, for example.
+     *
+     * Any custom implementation must be capable of sending
+     * [Error Reports](https://docs.bugsnag.com/api/error-reporting/)
+     * and [Sessions](https://docs.bugsnag.com/api/sessions/) as
+     * documented at [https://docs.bugsnag.com/api/](https://docs.bugsnag.com/api/)
+     */
+    var delivery: Delivery? = null
+
+    /**
+     * Set the endpoints to send data to. By default we'll send error reports to
+     * https://notify.bugsnag.com, and sessions to https://sessions.bugsnag.com, but you can
+     * override this if you are using Bugsnag Enterprise to point to your own Bugsnag endpoints.
+     */
+    var endpoints = Endpoints()
+
+    /**
+     * Set the maximum number of breadcrumbState to keep and sent to Bugsnag.
+     * By default, we'll keep and send the 25 most recent breadcrumb log
+     * messages.
+     */
+    var maxBreadcrumbs = DEFAULT_MAX_SIZE
+        set(numBreadcrumbs) {
+            field = when {
+                numBreadcrumbs <= MIN_BREADCRUMBS -> MIN_BREADCRUMBS
+                numBreadcrumbs > MAX_BREADCRUMBS -> MAX_BREADCRUMBS
+                else -> numBreadcrumbs
+            }
+        }
+
+    /**
+     * Set the context sent to Bugsnag. By default we'll attempt to detect the
+     * name of the top-most activity at the time of a report, and use this
+     * as the context, but sometime this is not possible.
+     */
+    var context: String? = null
+
+    /**
+     * Set which keys should be redacted when sending metadata to Bugsnag.
+     * Use this when you want to ensure sensitive information, such as passwords
+     * or credit card information is stripped from metadata you send to Bugsnag.
+     * Any keys in metadata which contain these strings will be marked as
+     * REDACTED when send to Bugsnag.
+     *
+     * For example:
+     *
+     * client.setRedactKeys("password", "credit_card");
+     */
+    var redactKeys: Set<String>
+        get() = Collections.unmodifiableSet(metadata.redactKeys)
+        set(redactKeys) = metadata.setRedactKeys(redactKeys)
+
+
+    var loggingEnabled: Boolean
```

Comment:
I suspect this should this go away after the logger changes have been combined?

Reply:
This is correct